File: app/src/main/java/com/octavianmetta/android/myanimelistsearcher/models/anime/characterStaff/VoiceActor.java

package com.octavianmetta.android.myanimelistsearcher.models.anime.characterStaff;

import android.databinding.BaseObservable;
import android.databinding.Bindable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.octavianmetta.android.myanimelistsearcher.BR;

public class VoiceActor extends BaseObservable {

    @SerializedName("mal_id")
    @Expose
    private Integer malId;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("image_url")
    @Expose
    private String imageUrl;
    @SerializedName("language")
    @Expose
    private String language;

    public Integer getMalId() {
        return malId;
    }

    public void setMalId(Integer malId) {
        this.malId = malId;
    }

    @Bindable
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
        notifyPropertyChanged(BR.name);
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getLanguage() {
        return language;
    }

    public void setLanguage(String language) {
        this.language = language;
    }

}

File: app/src/main/java/com/octavianmetta/android/myanimelistsearcher/models/anime/characterStaff/VoiceActor.kt
package com.octavianmetta.android.myanimelistsearcher.models.anime.characterStaff

import android.databinding.BaseObservable
import android.databinding.Bindable

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.octavianmetta.android.myanimelistsearcher.BR

data class VoiceActor(
        @SerializedName("image_url")
        val imageUrl: String,
        @SerializedName("language")
        val language: String,
        @SerializedName("mal_id")
        val malId: Int,
        @SerializedName("name")
        val name: String,
        @SerializedName("url")
        val url: String) : BaseObservable()